lib/go/thrift/multiplexed_processor.go

```diff
@@ -1,3 +1,5 @@
+// +build go1.7
```

Comment:
Maybe this file should be named `multiplexed_protocol_go17.go` or something like that to be consistent with `TMultiplexedProcessor` in `multiplexed_protocol.go`?